src/main/java/frc/robot/commands/BounceAuto.java

```diff
@@ -84,10 +87,10 @@ public BounceAuto(DriveTrain robotDrive, DriveCommand drivecommand) {
                 configForward);
                   
         System.out.println("DEBUG: Bounce path");
-        System.out.println("DEBUG: maxSpeed = " + maxSpeed + " maxAcceleration = " + maxAccel);
+        System.out.print("DEBUG: maxSpeed = " + maxSpeed + " maxAcceleration = " + maxAccel + " ");
         double pathTime = backTrajectory1.getTotalTimeSeconds() + backTrajectory2.getTotalTimeSeconds()
             + forwardTrajectory1.getTotalTimeSeconds() + forwardTrajectory2.getTotalTimeSeconds();
-        System.out.println("DEBUG: Bounce path time = " + pathTime);
+        System.out.println("Path time = " + pathTime);
```

Comment:
again. random change and not actually useful.

Reply:
Same reason as before. Makes the debug output more readable.